WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -82,6 +82,8 @@ class WPMainActivityViewModel @Inject constructor(private val appPrefsWrapper: A
     }
 
     fun onTooltipTapped() {
+        appPrefsWrapper.setMainFabTooltipDisabled(true)
```

Comment:
I don't want to block this PR, however, I'd consider adding a simple unit test for this behavior. Wdyt?